tests/cpp.test

```diff
@@ -60,6 +60,7 @@
 30051  nl_namespace-a.cfg                   cpp/nl-namespace.h
 
 30052  nl_try-a.cfg                         cpp/try-catch-nl.cpp
+30053  sp_catch_brace.cfg                   cpp/exception.cpp
```

Comment:
Reusing an existing test input is good, but it would be better if you also changed the existing input to have two spaces where `sp_catch_brace` is meant to trigger, and add two configs; one `force` and one `remove`. This way you ensure the rule is actually triggering where expected. Right now, because the input and output are the same, the test could be passing by accident.